client/src/js/modules/calendar/views/calendar.js

```diff
@@ -285,6 +295,7 @@ define(['marionette', 'backbone', 'collections/visits', 'templates/calendar/cale
             this.onScroll = _.debounce(this.onScroll, 100)
             this.collection = new Backbone.Collection()
             this.days = new Backbone.Collection()
+            this.timezone = app.options.get('timezone') ? app.options.get('timezone') : 'Europe/London'
```

Comment:
This timezone declaration needs to be in the DayItemView. The child view does not inherit the parent properties.
Also the DayItemView is the only place where the timezone is used in this module, so you can make it a local variable in its initialize method. That way it will have scope within the map functions.